domains.txt

```diff
@@ -649,16 +724,20 @@ streamrecruitment.co.uk
 strikeits.com
 surestart-recruitment.co.uk
 swanitrecruitment.com
+synergizecl.co.uk
 synergy-emea.co.uk
 syntaxconsultancy.com
 t2cuk.com
 t4ssolutions.co.uk
+tab-it-recruitment.co.uk
```

Comment:
TAB-IT Recruitment Solutions is run by developers and we believe this, along with our approach, sets us apart.

We are the IT recruitment partner of choice and one that goes that extra mile.